armory/eval/plot_poisoning.py

```diff
@@ -3,11 +3,6 @@
 """
 import json
 
-import logging
-
-logger = logging.getLogger("matplotlib")
-logger.setLevel(logging.INFO)
```

Comment:
Is there an alternative way to enable logging for external libraries like matplotlib when desired (like here)?

Reply:
yes, there is now full support for external libraries that use `logging`